ynput/OpenPype
# Use host's node uniqueness for instance id in new publisher

## Changelog Description

Instead of writing `instance_id` as parm or attributes on the publish instances we can, for some hosts, just rely on a unique name or path within the scene to refer to that particular instance. By doing so we fix #4820 because upon duplicating such a publish instance using the host's (DCC) functionality the uniqueness for the duplicate is then already ensured instead of attributes remaining exact same value as where to were duplicated from, making `instance_id` a non-unique value. 

## Additional info

I've now tested the Houdini, Maya and Fusion change and for me these seem to work.

I'm not entirely sure whether there are any other potential issues due to `instance_id` now not being a uuid generated by the new publisher for these instances.

## Testing notes:

1. Create a publish instance in Maya, Fusion, Houdini
2. Duplicate the publish instance node inside the host (e.g. ROP node in Fusion)
3. Refresh the publisher - both the old and duplicated entry should show up and publish fine.
4. Upon changing attributes e.g. on the instance in the Publisher UI and saving that the changed values should still persist on the node in the workfile (even when altering on duplicated instances)
